generate.sh

```diff
@@ -38,7 +38,18 @@ function initrepo() {
   fi
 
   pushd $PROTO_DIR
-  proto_files=$(find . -name '*.proto' -not -name $EXCLUDE_PROTOS)
+  proto_files=$(find . -name '*.proto' -not -name $EXCLUDE_PROTOS | tr '\n' ' ')
```

Comment:
```suggestion
  proto_files=$(find . -name '*.proto' -not -path "$EXCLUDE_PROTOS")
```

Since loop is the only repo that uses the `EXCLUDE_PROTOS` var, we can safely modify its value to represent a path instead of a name.